Mathlib/NumberTheory/LSeries/DirichletContinuation.lean

```diff
@@ -67,4 +70,85 @@ lemma differentiable_LFunction {χ : DirichletCharacter ℂ N} (hχ : χ ≠ 1)
     Differentiable ℂ (LFunction χ) :=
   (differentiableAt_LFunction _ · <| Or.inr hχ)
 
+/-!
+## Results on changing levels
+-/
+
+lemma continuous_cpow_natCast_neg (n : ℕ) [NeZero n] : Continuous fun s : ℂ ↦ (n : ℂ) ^ (-s) :=
+  Continuous.const_cpow continuous_neg (.inl <| NeZero.ne (n : ℂ))
```

Comment:
Move to `Mathlib.Analysis.SpecialFunctions.Pow.Continuity` and add the following to `Mathlib.Analysis.SpecialFunctions.Pow.Deriv` ?

```lean
lemma differentiable_cpow_natCast_neg (n : ℕ) [NeZero n] : Differentiable ℂ fun s : ℂ ↦ (n : ℂ) ^ (-s) :=
  Differentiable.const_cpow differentiable_neg (.inl <| NeZero.ne (n : ℂ))
```

Reply:
I've added it to the upstream files. I'm not convinced about `1 - c * n ^ (-s)`, I had a look for where the new lemma could be used and the examples I found mostly weren't of that shape.

Reply:
The cases where these lemmas are applied are actually all about `1 - c * n ^ (-s)`.
But maybe a better approach is to teach `fun_prop` to use the two new lemmas; then these three proofs should reduce to just `fun_prop`.